fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/Expression.java

```diff
@@ -45,8 +45,8 @@
  * Abstract class for all Expression in Nereids.
  */
 public abstract class Expression extends AbstractTreeNode<Expression> implements ExpressionTrait {
-
-    private static final String INPUT_CHECK_ERROR_MESSAGE = "argument %d requires %s type, however '%s' is of %s type";
+    // Mask this expression is generated by rule, should be removed.
+    public boolean isGeneratedIsNotNull = false;
```

Comment:
should we process this attr in each withXXX function? i think we will lose this tag after rewrite.